test/org/sagebionetworks/bridge/services/UserAdminServiceMockTest.java

```diff
@@ -149,9 +152,7 @@ public void creatingUserConsentsToAllRequiredConsents() {
         assertEquals(participant.getEmail(), signIn.getEmail());
         assertEquals(participant.getPassword(), signIn.getPassword());
         
-        for (SubpopulationGuid guid : session.getConsentStatuses().keySet()) {
-            verify(consentService).consentToResearch(eq(study), eq(guid), any(StudyParticipant.class), any(), eq(SharingScope.NO_SHARING), eq(false));
-        }
```

Comment:
Why did we remove this from the test? Does creating a user with the consentUser no longer function?

Reply:
I see no reason, I've restored it and updated the tests to pass (the method was itself reorganized).